src/pages/editor/EditorInstitution.tsx

```diff
@@ -61,63 +62,107 @@ export const EditorInstitution = () => {
+            <Grid item xs={4} md={6}>
+              <Typography variant="h3" component="h2">
+                {t('editor.institution.institution_short_name')}
+              </Typography>
+              <Typography>{organizationQuery.data?.acronym ?? '-'}</Typography>
+            </Grid>
+
+            <Grid item xs={4} md={6}>
+              <Typography variant="h3" component="h2">
+                {t('editor.institution.institution_code')}
+              </Typography>
+              <Typography>{institution?.id.split('/').pop() ?? '-'}</Typography>
+            </Grid>
+          </Grid>
+
+          <Grid item xs={12} md={4}>
+            <Typography variant="h3" component="h2">
+              {t('basic_data.institutions.sector')}
+            </Typography>
+            <Typography>
+              {customer?.sector ? t(`basic_data.institutions.sector_values.${customer.sector}`) : '-'}
+            </Typography>
+          </Grid>
+
+          <Grid item xs={12} md={4}>
+            <Typography variant="h3" component="h2">
+              {t('editor.institution.unique_feide_id')}
+            </Typography>
+            <Typography>{customer?.feideOrganizationDomain ?? '-'}</Typography>
+          </Grid>
+
+          <Grid item xs={12} md={4}>
+            <Typography variant="h3" component="h2">
+              {t('basic_data.institutions.ror')}
+            </Typography>
+            <Typography>
+              {customer?.rorId ? (
+                <Link href={customer.rorId} target="_blank" rel="noopener noreferrer">
+                  {customer.rorId}
+                </Link>
+              ) : (
+                '-'
+              )}
+            </Typography>
+          </Grid>
+
+          <Grid item xs={12}>
+            <Typography variant="h3" component="h2">
+              {t('common.nvi')}
+            </Typography>
+            <Typography>
+              {customer?.nviInstitution
+                ? t('editor.institution.institution_is_nvi_institution')
+                : t('editor.institution.institution_is_not_nvi_institution')}
+            </Typography>
+          </Grid>
+
+          <Grid item xs={12}>
+            <Typography variant="h3" component="h2">
+              {t('common.rbo')}
+            </Typography>
+            <Typography>
+              {customer?.rboInstitution ? t('editor.institution.rbo_funded') : t('editor.institution.not_rbo_funded')}
+            </Typography>
+          </Grid>
+
+          <Grid item xs={12}>
+            <Typography variant="h3" component="h2">
+              {t('editor.institution.institution_support')}
+            </Typography>
+            <Link href={customer?.serviceCenterUri} target="_blank" rel="noopener noreferrer">
+              {customer?.serviceCenterUri}
+            </Link>
+          </Grid>
 
           {institutionUsers && (
-            <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: '3rem' }}>
+            <Grid container item xs={12} md={8}>
```

Comment:
Subjektivt forslag igjen

```suggestion
            <Grid container item xs={12}>
```